impeller/compiler/reflector.cc

```diff
@@ -767,6 +773,75 @@ std::vector<StructMember> Reflector::ReadStructMembers(
       continue;
     }
 
+    // Tightly packed half Point (vec2).
+    if (member.basetype == spirv_cross::SPIRType::BaseType::Half &&  //
+        member.width == sizeof(float) * 4 &&                         //
```

Comment:
Can you double check this? The member width doesn't make sense for half base types. I may be misunderstanding.

Reply:
ah, this was a hack I left in. Should be `sizeof(Half) * 8`